src/content/pages/explore.mdx

```diff
@@ -6,136 +6,129 @@ subtitle: At EuroPython, many attendees will come with their families. While not
 import { Image } from "astro:assets";
 import venueImage from "./images/prague.jpg";
 
-# More Tips on Exploring Prague
+# Prague Exploration Tips
 
 <Image src={venueImage} width="1200" alt="" />
 
-<br></br>
+<div class="mb-15"> </div>
 
-At EuroPython, many attendees will come with their families. While not attending the conference, why not go and explore Prague and the surrounding area?
+At EuroPython, many attendees come with their families and friends. When you're not attending the conference, why not explore Prague and its surrounding areas?
 
-Here are some tips and things to do provided by our volunteers. We hope they help you and your companions have a great time in Prague:
+Here are some tips and recommendations from our volunteers. We hope they help you and your companions have a great time in Prague!
 
-## Traveling around
+## Getting around
 
 ### Public transport
 
-In Prague, the recommended way to get around is public transport:
+In Prague, the recommended way to get around is by public transport:
 
-- There’s some info [here](https://ep2024.europython.eu/where#getting-around-in-prague)
-- You can use your normal card to travel around by train within Prague
-- It is usually fast and reliable
-- After midnight, the metro stops, but there are buses
-- Mostly safe, with no no-go stations or anything similar
+- More information can be found [here](https://ep2024.europython.eu/where#getting-around-in-prague).
+- You can use your regular card to travel around by train within Prague.
```

Comment:
I don't know. Maybe it is "regular passes"?

Reply:
yes, regular passes - basically daily pass, weekly pass etc.

Reply:
I think it should be "credit card". I used to just tap my credit card to buy a ticket.

Reply:
Ah that makes sense. But I am confused here. Do you really need to take trains to travel around Prague? Most of the information I read says you need a train if you want to travel in and our of Prague. Within Prague, it's buses, trams, metros. Does this maybe mean `You can use your regular bank cards to travel by train to the areas surrounding Prague."`?

Reply:
Oh, my brain automatically ignored it and changed train to metro 😅

In this case I'd say "public transport", and maybe insert a link that points to a page about public transport in Prague 🙂

Reply:
ahaha what a magical brain! I think I will just do this:
- More information on getting around by public transport can be found [here](https://ep2024.europython.eu/where#getting-around-in-prague).
- You can use your regular bank cards for public transport within Prague.

Since I don't want to link the same thing twice here.

Reply:
Applied in https://github.com/EuroPython/website/pull/748/commits/d5170535912668ad3f56ff2015c1ed834a94e3c7